README.md

```diff
@@ -60,27 +58,43 @@
 |---------------------------|----------------------------|
 |getOpportunities           | obtainWonOpportunityList   |
 
-
 ## 4. Constants naming
 
 * Constants (i.e. variables defined as static final) should be in uppercase with words separated by underscores:
+
 ```java
 public static final String PAYPAL_LOGIN_URL = 'https://login.paypal.com/';
 ```
 
 ## 5. SOQL and SOSL queries
 
-* SOQL and SOSL commands should be in uppercase:
+* SOQL and SOSL keywords should be uppercase.
+* Fields to retrieve should be on separate lines, with the separating comma on the same line as the field.
```

Comment:
This is only applicable when there are several fields in the Query. When the query is retrieving just a few fields, than its preferable to have it in the same line.

Reply:
I've modified the text a bit to reflect small and bigger SOQL statements.